src/test/java/com/carmen/app/control/CarServiceTest.java

```diff
@@ -52,16 +55,16 @@ public void setUp() throws Exception {
 
 	@Test
 	public void testCarCreated() {
-		doNothing().when(this.persistenceService).createNew(car);
+		when(this.persistenceService.createNew(car)).thenReturn(car);
 		assertEquals(car, carService.createCar(car));
 		verify(this.persistenceService).createNew(car);
 	}
 
 	@Test
 	public void testGetCars() {
```

Comment:
Its a good idea to test corner cases of your method. Usually values like NULL, 0, negative numbers or very large numbers are corner cases to check, and unit tests is a nice way to do it.